extensions/wikia/Search/UnifiedSearchIndexingController.class.php

```diff
@@ -49,36 +49,52 @@ private function findResults( array $namespaces, $offset, $limit ) {
 				->LIMIT( $limit + 1 )// fetching one more than limit to get next offset
 				->ORDER_BY( [ 'page.page_id', 'asc' ] )
 				->runLoop( $db, function ( &$result, $row ) {
-					$result[] = $row->page_id;
+					$result[] = [ 'id' => $row->page_id ];
 				} );
 
 		$splitByLimit = array_chunk( $results, $limit );
 
+		$items = isset($splitByLimit[0]) ? $splitByLimit[0] : [];
+
 		return [
-			'nextOffset' => isset( $splitByLimit[1][0] ) ? $splitByLimit[1][0] : null,
-			'pageIds' => isset( $splitByLimit[0] ) ? $splitByLimit[0] : [],
+			'nextOffset' => isset( $splitByLimit[1][0]['id'] ) ? $splitByLimit[1][0]['id'] : null,
+			'pageIds' => array_map(function (array $item) {
```

Comment:
Should be spacey per coding conventions:
```suggestion
			'pageIds' => array_map( function ( array $item ) {
```

Reply:
Sorry, I will configure my IDE ;)